src/tests/GfxReplayTest.cpp

```diff
@@ -465,6 +476,107 @@ void GfxReplayTest::testSimulatorIntegration() {
   }
 }
 
+// test lights data by recording and playback through the simulator interface
+void GfxReplayTest::testLightIntegration() {
+  const auto compareLightSetups = [&](const LightSetup& a,
+                                      const LightSetup& b) {
+    for (int i = 0; i < a.size(); ++i) {
+      CORRADE_COMPARE(a[i].vector, b[i].vector);
+      CORRADE_COMPARE(static_cast<int>(a[i].model),
+                      static_cast<int>(b[i].model));
+      CORRADE_COMPARE(a[i].color, b[i].color);
+    }
+  };
+
+  const auto testFilepath =
+      Corrade::Utility::Path::join(DATA_DIR, "./gfx_replay_test.json");
+  const LightInfo pointLight0{Magnum::Vector4{1.5f, 2.0f, 2.5f, 1.0f},
```

Comment:
```suggestion
  const LightInfo pointLight0{{1.5f, 2.0f, 2.5f, 1.0f},
```

should work too I think, same below